event_registration_multi_qty/models/event.py

```diff
@@ -77,3 +77,13 @@ def _check_attendees_qty(self):
                     _('You can not add quantities if you not active the'
                       ' option "Allow multiple attendees per registration"'
                       ' in event'))
+
+    @api.model
+    def _prepare_attendee_values(self, registration):
+        res = super()._prepare_attendee_values(registration)
+        # Passed fields are not taken into account if a default is set.
+        # We can force this with a specific context
+        qty = self.env.context.get("default_qty")
```

Comment:
Why not to check `registration.get("qty")` instead?

Reply:
Ok, it's just that, in that case, we would be completely skipping Odoo's mechanism applied in the mentioned commit. Maybe that was a necessary fix for some use case and we might run into problems in the future? This way, we would avoid changing a lot of the current behavior...

Reply:
It's not a change of the behavior. It's expected that this field doesn't default unless it's not informed. So if a method aks to register 5 attendees who are we to state the contrary :sweat_smile:

Reply:
Ok, makes sense :smiley: Changes made!